app/serializers/v3/product_serializer.rb

```diff
@@ -25,7 +25,7 @@ def arch
   end
 
   def eula_url
-    if object.eula_url
+    if object.eula_url.to_s != ''
```

Comment:
`object.eula_url.present?` should do it.

Reply:
You're right, I pushed a fix